packages/core/src/utils/signEvent.ts

```diff
@@ -9,9 +9,12 @@ export const signEvent = async <T extends EventBase>(
 	event: T,
 	signature: SigningKey,
 	signingName: string,
+	prune = true,
```

Comment:
I'd say it would be better to prune it outside of this function, I don't it's a good a idea to provide a boolean to prune it here

Reply:
For now did it like to to avoid breaking other tests. 

This will be tackled in the item i mentioned in the channel - using persistenteventbase everywhere. In my radar next.